backend/app/moderation.py

```diff
@@ -522,7 +520,7 @@ def submit_review(
         category = EmailCategory.MODERATION_REJECTED
         subject = f"Change in build #{request.build_id} rejected"
 
-        if not verification_status.verified:
+        if models.DirectUploadApp.by_app_id(sqldb, request.appid) is not None:
```

Comment:
Check for an empty list https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Result.all

Reply:
Yeah, that's some brain fart from programming in the early morning. Fixed